pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -120,14 +120,14 @@ let
          null)
     else "";
 
-  parseResponseFile = if buildPackages.stdenv.cc or null != null && buildPackages.stdenv.cc != "/dev/null"
+  expand-response-params = if buildPackages.stdenv.cc or null != null && buildPackages.stdenv.cc != "/dev/null"
   then buildPackages.stdenv.mkDerivation {
-    name = "parse-response-file";
-    src = ./parseResponseFile.c;
+    name = "expand-response-params";
+    src = ./expand-response-params.c;
     buildCommand = ''
       # Make sure the output file doesn't refer to the input nix path
-      cp "$src" parseResponseFile.c
-      "$CC" -O3 -o "$out" parseResponseFile.c
+      cp "$src" expand-response-params.c
+      "$CC" -O3 -o "$out" expand-response-params.c
```

Comment:
Free free to try it with `$src` and investigate the error on Darwin! `__FILE__` is only a guess.